### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Daniil-A4**
- [DEMO LINK](https://Daniil-A4.github.io/layout_catalog/)
- [TEST REPORT LINK](https://Daniil-A4.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,8 +5,300 @@
...
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <a href="index.html" class="header__logo">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+        <nav class="nav">
```
Comment (Esceype) [inline src/index.html:19]: Add one empty line here

File: src/index.html [hunk 2]
```diff
@@ -5,8 +5,300 @@
...
+            </li>
+
+            <li class="nav__item">
+              <a href="#video" class="nav__link">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main>
```
Comment (Esceype) [inline src/index.html:62]:
```suggestion
    </header>
    
    <main>
```

File: src/index.html [hunk 3]
```diff
@@ -5,8 +5,300 @@
...
+      <div class="cards">
+        <div class="cards__card card" data-qa="card">
+          <img
+            class="card__img"
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+          >
+          <h3 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
```
Comment (Esceype) [inline src/index.html:70]:
```suggestion
          >
          <h3 class="card__title">
            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
          </h3>
```

File: src/styles/cards.scss [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+  @media (max-width: 767px) {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  @media (max-width: 487px) {
+    grid-template-columns: repeat(1, 1fr);
+  }
+
+  .card {
```
Comment (Esceype) [inline src/styles/cards.scss:23]: To separate file if this is another one BEM block

File: src/index.html [hunk 5]
```diff
@@ -5,8 +5,300 @@
...
+
+          <div class="card__box">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__review">Reviews: 5</p>
```
Comment (Esceype) [inline src/index.html:82]:
```suggestion
            </div>
            
            <p class="card__review">Reviews: 5</p>
```

File: src/index.html [hunk 6]
```diff
@@ -5,8 +5,300 @@
...
+          </div>
+
+          <div class="card__box card__box--retreat">
+            <p class="card__price-text">Price:</p>
+            <p class="card__price">$2,199</p>
+          </div>
+
+          <a
+            href="#"
```
Comment (Esceype) [inline src/index.html:91]:
```suggestion
            href="#buy"
```

File: src/styles/header.scss [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+}
+
+.header__logo {
+  display: flex;
+}
+
+.nav {
```
Comment (Esceype) [inline src/styles/header.scss:19]: To separate file

**Review by Esceype — CHANGES_REQUESTED** [review]
GJ!!! Overall move files to with reset and vars to utils folder inside styles and blocks files to blocks folder inside styles folder

File: src/styles/main.scss [hunk 8]
```diff
@@ -1,3 +1,12 @@
 body {
   margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
 }
```
Comment (denys-danyliuk) [inline src/styles/main.scss:4]: move it to other file

File: src/styles/blocks/card.scss [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+    &:hover {
+      background: #fff;
+      color: $accentColor;
+    }
+  }
+}
+
+.card:hover .card__title {
+  color: #34568b;
+}
```
Comment (denys-danyliuk) [inline src/styles/blocks/card.scss:96]: don't increase specificity, also use sass nesting, 10'th point in checklist

File: src/styles/blocks/header.scss [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+}
+
+.header__logo {
+  display: flex;
+}
```
Comment (denys-danyliuk) [inline src/styles/blocks/header.scss:17]: 10'th point in checklist

File: src/styles/blocks/cards.scss [hunk 11]
```diff
@@ -0,0 +1,22 @@
+.cards {
+  margin: 0 auto;
```
Comment (denys-danyliuk) [inline src/styles/blocks/cards.scss:2]: use mix, 8'th point in checklist

**Review by denys-danyliuk — CHANGES_REQUESTED** [review]

**Review by Esceype — APPROVED** [review]
Great job!!!

**Pull request closed without merging**